gulp/tasks/watch.js

```diff
@@ -4,8 +4,12 @@
 const gulp = require( 'gulp' );
 const config = require( '../config' );
 
-gulp.task( 'watch', [ 'browsersync' ], function() {
-  gulp.watch( config.scripts.src, [ 'scripts' ] );
-  gulp.watch( config.styles.cwd + '/**/*.less', [ 'styles' ] );
-  gulp.watch( config.images.src, [ 'images' ] );
-} );
+gulp.task( 'watch',
+  gulp.series(
+    'browsersync', () => {
```

Comment:
Should the anonymous arrow function here be on the next line?

Reply:
Makes sense. Moved it to the next line.